File: Listas de Exercícios (C)/C/Lista 2 - Sala/6.c
#include <stdio.h>
#include <math.h>

int main()
{
    float A, B, C, D, X1, X2;
    
    printf("Equação de 2º grau\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("A: "); scanf("%f", &A);
    printf("B: "); scanf("%f", &B);
    printf("C: "); scanf("%f", &C);
    
    printf("\n");
    
    D=pow(B, 2.0)-4*A*C;
    
    if (D<0) {
        printf("Ambas as raízes da equação não são números Reais!");
    }
    else if (D==0) {
        printf("Ambas as raízes da equação são Reais e iguais!");
    }
    else {
        printf("Ambas as raízes são Reais e distintas!");
    }
    
    printf("\n");
    
    D=pow(D, 0.5);
    
    X1=(-B+D)/2*A;
    printf("Raíz 1: %.2f\n", X1);
    
    X2=(-B-D)/2*A;
    printf("Raíz 2: %.2f\n", X2);
}

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/Python/Lista 2 - Sala/6.py
print("Equação de 2º grau\n")

print("--Valores decimais devem ser digitados com (.)!--")

A=float(input("A: "))
B=float(input("B: "))
C=float(input("C: "))

print("")

D=B**2-4*A*C

if (D<0):
    print("Ambas as raízes da equação não são números Reais!")
elif (D==0):
    print("Ambas as raízes da equação são iguais!")
else:
    print("Ambas as raízes da equação são Reais e distintas!")

X1=(-B+D**0.5)/2*A
print(f"Raíz 1: {X1:.2f}")

X2=(-B-D**0.5)/2*A
print(f"Raíz 2: {X2:.2f}")

print("")